app/components/elements/TextLink.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
This file is not really required.

The `TextLink` could just be imported and used directly from `FontStyles`